Project-2/Jvonderh Project 2.md

```diff
@@ -0,0 +1,148 @@
+# Project 2
+
+Your second project will require you to answer each of the 10 questions below.  You will be expected to open a pull request with your initial answers by the second class meeting, giving you one week to work on these problems. You and your peers will then have one week to work together to refine your respective initial answers, so they are ready for final submission. Once your pull requests have been reviewed and merged to the development branch, I will review them, then merge to the master branch. 
+
+```
+Tip #1: Carefully study the Baader, et. al. selections assigned on bisimulation; it is deceptively subtle, and quite powerful. 
+Tip #2: Google is still your friend. So is stackexchange...
+Tip #3: Work together to solve these problems, even for initial submissions and when you do, document this in github. 
+Tip #4: Work together as a team. 
+```
+
+1. Let V be a vocabulary of ALCI consisting of a role name "P". Interpret part_of as "x is a part of y". Using this role name, define the following formulas in this language:
+```
+  (a)  PP that says that x is a proper part of y
+  PP= ∃part_of^~∃part-of-
+  (b)  iPP that says that y is a proper part of x
+  iPP= ∃part_of-^~∃part-of
```

Comment:
So 'x is proper part of y' and 'y is proper part of x' are equivalent? Come on now...